data/spider-20190205/generate.sh

```diff
@@ -1,36 +0,0 @@
-#!/bin/bash
```

Comment:
We need this file (and other existing files in data/spider-20190205) to regenerate the data used for previous experiments.

Reply:
Ok, I'll keep them in for now. I deleted them because I made `data/spider-20190205` a symlink to the actual data directory (including these files), but if all our configs allow the data directory to be specified, that solves the issue better.